modules/libcom/src/as/asLib.h

```diff
@@ -29,6 +29,8 @@ struct dbChannel;
  */
 LIBCOM_API extern int asCheckClientIP;
 
+typedef struct asIdentity *ASIDENTITYPVT;
```

Comment:
Please add a feature macro to allow API users to clearly identify when the new definitions are present.  eg.

```c
/** Identifies added API associated with asAddClientIdentity()
 * @since UNRELEASED
 */
#define EPICS_ASLIB_HAS_IDENTITY
```

Note that this macro is practically redundant to a test for `S_asLib_dupMethod`.